layout/layout.h

```diff
@@ -171,7 +171,13 @@ enum {
 	M0_LAYOUT_ENUM_TYPE_MAX = 32
 };
 
-enum { M0_DEFAULT_LAYOUT_ID = 1 };
+#ifdef CONFIG_X86_64
+	enum { M0_DEFAULT_LAYOUT_ID = 1 };
+#elif defined CONFIG_AARCH64
+	enum { M0_DEFAULT_LAYOUT_ID = 5 }; /* RHEL pages size on aarch64 : 64KB */
```

Comment:
No need to mention RHEL here.

Reply:
According to Google, looks like it's [pertinent to RHEL](https://news.ycombinator.com/item?id=27513209). Other distributions may have different page sizes on ARM64. (For example, [Ubuntu has 4K pages on ARM64](https://wiki.ubuntu.com/ARM64/performance).)

Reply:
Got following info from @andriytk , seems Keeping RHEL is fine.
    
Ubuntu currently supports 4k page size for all architectures except for ppc64el.

https://wiki.ubuntu.com/ARM64/performance
​RHEL used 64k page size on aarch64 for a while

Reply:
I think more commentary is needed here. How layout identifiers are related to page size?